src/main/java/com/gbtec/Cell.java

```diff
@@ -0,0 +1,18 @@
+package com.gbtec;
+
+public class Cell {
+
+    private boolean isAlive;
+
+    public Cell() {
+        isAlive = false;
```

Comment:
A "boolean" is per default set to false ... so we don´t need to set it here explicitly. Also the whole constructor is here unnecessary.